.pre-commit-config.yaml

```diff
@@ -1,27 +1,29 @@
 ---
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v3.1.0
+    rev: v4.4.0
     hooks:
       - id: trailing-whitespace
       - id: check-added-large-files
       - id: check-docstring-first
       - id: check-merge-conflict
       - id: end-of-file-fixer
   - repo: https://github.com/ambv/black
-    rev: 21.12b0
+    rev: 22.12.0
     hooks:
       - id: black
         language_version: python3.7
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.910-1
+    rev: v0.991
     hooks:
       - name: Run mypy static analysis tool
         id: mypy
         args: [--no-warn-unused-ignores, --ignore-missing-imports]
         files: src
-  - repo: https://github.com/flakehell/flakehell/
-    rev: v.0.9.0
+        additional_dependencies:
+          - pydantic
```

Comment:
I had to add the additional dependency "pydantic" here, because it was not present on my machine, but was necessary to run the pre-commit-hook.

See also the discussion here: https://gitter.im/tiangolo/fastapi?at=5dfcda8844e1fb33f6fc13d2